scripts/deploy-local.js

```diff
@@ -4,21 +4,21 @@ const { deployBridge } = require("../lib/bridge");
 const { prepareToucanEnv } = require("../lib/toucan");
 
 async function deploy() {
-	const [owner] = await hre.ethers.getSigners();
-	const regenBridgeAccount = "0x11241e35B3f79099123aA0C1C4c97b1FcdCd21f6";
```

Comment:
why was this actually hardcoded before?

Reply:
Not 100% sure why I hardcoded it... I don't think I know much about how it was working so I think I assumed that account was somehow needed... but definitely wasn't needed to be that particular account.

Reply:
cool, got it!